lib/vmdb/initializer.rb

```diff
@@ -12,8 +12,10 @@ def self.init
       #   * command line(rails server)
       #   * debugger
       if defined?(Rails::Server)
-        MiqUiWorker.preload_for_worker_role
+        # preload_for_worker_role depends on seeding, principally MiqDatabase
```

Comment:
Based on this comment, should the Rails console bit down below be changed as well?